lib/api_repo.js

```diff
@@ -71,13 +72,13 @@ function ApiRepo(opts) {
   this.languages = opts.languages || DEFAULT_LANGUAGES;
   this.outDir = opts.outDir || tmp.dirSync().name;
   this.gaxDir = opts.gaxDir;
-  this.repoDir = opts.repoDir;
+  this.repoDirs = opts.repoDirs || [];
   this.templateRoot = opts.templateRoot;
   this.pkgPrefix = opts.pkgPrefix;
   this.overridePlugins = opts.overridePlugins;
   this.zipUrl = opts.zipUrl;
   this.isGoogleApi = !!opts.isGoogleApi;
-  if (!(this.repoDir || this.zipUrl)) {
+  if (!(this.repoDirs.length || this.zipUrl)) {
```

Comment:
I like to be more descriptive -- please compare exactly like `this.repoDirs.length === 0`, or use lodash like `_.isEmpty(this.repoDirs)` (see https://lodash.com/docs#isEmpty).